recipes/qdldl-python/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.1.3" %}
+
+package:
+  name: qdldl-python
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/q/qdldl/qdldl-{{ version }}.tar.gz
+  sha256: 2e98bfcc43de55225b82f5895f8266e79a305b174490c60bc559c1538f1be920
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+    - numpy
+    - pybind11
+    - scipy
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+
+test:
+  requires:
+    - pytest
+  imports:
+    - qdldl
+  source_files:
+    - tests/
+  commands:
+    - pytest tests
+
+about:
+  home: https://github.com/oxfordcontrol/qdldl-python
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: 'Python interface to the QDLDL free LDL factorization routine for quasi-definite linear systems'
+
+  description: |
+    Python interface to the [QDLDL](https://github.com/oxfordcontrol/qdldl/) free
+    LDL factorization routine for quasi-definite linear systems: `Ax = b`.
+  dev_url: https://github.com/oxfordcontrol/qdldl-python
+
+extra:
+  recipe-maintainers:
+    - bstellato
+    - gbanjac
```

Comment:
@gbanjac - are you OK with being a maintainer?

Reply:
Yes, sure.